src/libs/test.sh

```diff
@@ -11,6 +11,5 @@ reldir=`realpath --relative-to $topdir .` \
 
 start libs/assay_type
 CMD="python ./assay_type.py"
-diff ${topdir}/examples/libs/assay_type_out.txt <( eval $CMD ) \
-  || die "Try: python ${reldir}/assay_type.py > examples/libs/assay_type_out.txt"
-end libs/assay_type
\ No newline at end of file
+eval $CMD || die "The doctest entries in ${reldir}/assay_type.py need to be updated"
```

Comment:
```suggestion
python ./assay_type.py
```
I think the failure message from doctest is sufficient: Not sure if wrapping it adds value.

Reply:
I'm going to claim that it does add value- I know it does for me, because I'm not used to doctest, and I think it would for Bill.  So I'll leave this one as it is.